app/institutions/discover/controller.ts

```diff
@@ -1,7 +1,37 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
 import CurrentUser from 'ember-osf-web/services/current-user';
+import { tracked } from '@glimmer/tracking';
+import Media from 'ember-responsive';
+import { action } from '@ember/object';
+import pathJoin from 'ember-osf-web/utils/path-join';
+import config from 'ember-get-config';
 
 export default class InstitutionDiscoverController extends Controller {
     @service currentUser!: CurrentUser;
+    @service media!: Media;
+
+    @tracked q?: string = '';
+    @tracked sort?: string =  '-relevance';
+    @tracked resourceType?: string = 'All';
+
+    queryParams = ['q', 'page', 'sort', 'resourceType'];
+
+    get showSidePanelToggle() {
+        return this.media.isMobile || this.media.isTablet;
+    }
+
+    get defaultQueryOptions() {
+        return {
+            resourceType: 'osf:Institutions',
+            publisher: pathJoin(config.OSF.url, 'institutions'),
```

Comment:
We don't need `resourceType` here, as users should be able to pick and choose resourceType in this view.
I think the `publisher` option needs the id at the end like:
```suggestion
            publisher: pathJoin(config.OSF.url, 'institutions', this.model.id),
```

Reply:
Got it, this is now updated.

Reply:
@futa-ikeda , this breaks the build when adding your commit suggestion. I agree with the general concept, but if there is a more complete answer, I can update that instead.

Reply:
I tried this locally and it doesn't break the build for me. What are the errors?

Reply:
It was resulting in a 404 on Chrome. This is now updated and working.